terraform-beyond-layer0/example-1/modules/guestbook_service/main.tf

```diff
@@ -0,0 +1,59 @@
+#
+# modules/guestbook_service/main.tf
+#
+
+resource "layer0_load_balancer" "guestbook" {
+  name        = "${var.load_balancer_name}"
+  environment = "${var.layer0_environment_id}"
+
+  port {
+    host_port      = 80
+    container_port = 80
+    protocol       = "http"
+  }
+}
+
+resource "layer0_service" "guestbook" {
+  name          = "${var.service_name}"
+  environment   = "${var.layer0_environment_id}"
+  deploy        = "${layer0_deploy.guestbook.id}"
+  load_balancer = "${layer0_load_balancer.guestbook.id}"
+
+  #scale = 3
```

Comment:
The guide references this and asks the user to uncomment the scale property to increase the scale. https://github.com/quintilesims/layer0/pull/184/files#diff-f672345b0c16fae2a231ebbed42c512cR403